packages/hub/src/lib/cache-management.ts

```diff
@@ -16,12 +16,19 @@ function getHuggingFaceHubCache(): string {
 	return process.env["HUGGINGFACE_HUB_CACHE"] ?? getDefaultCachePath();
 }
 
-function getHFHubCache(): string {
+export function getHFHubCache(): string {
```

Comment:
If it's exported, let's rename to

```suggestion
export function getHFHubLocalCachePath(): string {
```

Reply:
Renamed, you are right, this is more explicit and easier to understand.